pkg/deploy/config.go

```diff
@@ -101,6 +101,10 @@ type Configuration struct {
 	SubscriptionResourceGroupLocation  *string                `json:"subscriptionResourceGroupLocation,omitempty" value:"required"`
 	VMSize                             *string                `json:"vmSize,omitempty" value:"required"`
 	VMSSCleanupEnabled                 *bool                  `json:"vmssCleanupEnabled,omitempty"`
+	RPVmssIpTags                       []IPTag                `json:"rpVmssIpTags,omitempty"`
+	RPVmssIpTagsDisabledRegions        []string               `json:"rpVmssIpTagsDisabledRegions,omitempty"`
```

Comment:
I've never considered it until now but could we do better than strings for regions i.e. enums. Outside the scope of this PR but I came across this: https://gist.github.com/ausfestivus/04e55c7d80229069bf3bc75870630ec8

What do you think?